provd/proto/theme.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package theme;
+
+service ThemeService {
```

Comment:
Again, split these into individual calls so that they can be called independently and the user can watch the effects take place.